examples/get-started/first_figure.py

```diff
@@ -131,7 +130,34 @@
     land="lightgreen",
     water="lightblue",
     projection="M10c",
-    frame=True,
+    frame="a",
+)
+fig.show()
+
+###############################################################################
+# Add a title
+# -----------
+#
+# The ``frame`` parameter can be used to add a title to the figure. The title
+# is set with by passing ``"+t"`` followed by the title (e.g. setting the map
+# title to "Title" would be ``"+tTitle"``).
+#
+# To pass multiple arguments to ``frame``, a list can be used, as shown in the
+# example below. This format uses ``frame`` to set both the axes gridlines and
+# the figure title.
+#
+# If the figure title has multiple words, the string to set the title needs to
+# use single-quotes, with the actual title set in double quotes (e.g. setting
+# the title to "Coastal Maine" would use the syntax ``'+t"Coastal Maine"'``.
```

Comment:
Feel like this could be worded a bit better. Something along the lines of double quotes nested inside single quotes?

Or maybe I should finish up #1487 😂

Reply:
I think "nested" might be a confusing term for someone not familiar with Python terms.

Reply:
Ok, how about something like this?
```suggestion
# be wrapped in single-quotes, while the actual title is set in double quotes
# (e.g. setting the title to "A Title" would use the syntax ``'+t"A Title"'``.
```